charts/invenio/templates/nginx-configmap.yaml

```diff
@@ -187,7 +187,7 @@ data:
         # Another location is defined in order to allow large file uploads in the files
         # API without exposing the other parts of the application to receive huge
         # request bodies.
-        location ~ /api/records/.+/draft/files/.+/content {
+        location ~ /api/records/.+/draft/files/.+/content(/.+)?$ {
```

Comment:
Question: Do we risk DoS if unlimited deeply nested paths like `/content/foo/bar/...` are allowed with large payloads?
Suggest matching only expected patterns `/content or /content/part/<number>` to reduce attack surface.
Instead of:
`location ~ /api/records/.+/draft/files/.+/content(/.+)?$ {`
Use:
`location ~ ^/api/records/.+/draft/files/.+/content(/part/[0-9]+)?/?$ {`

Reply:
Basically, it must bet match these two routes & nothing else:
```
https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/resources/files/config.py#L26-L27
```

Reply:
I'll update the PR & also related PR https://github.com/inveniosoftware/cookiecutter-invenio-rdm/pull/309 & docs